bodyprocessors/json_test.go

```diff
@@ -73,50 +95,66 @@ func TestJSONToMap(t *testing.T) {
             ]
         ]
     }
-]
-`
-	)
-	mapAsserts := map[string]string{
-		"json.a":         "1",
-		"json.b":         "2",
-		"json.c":         "3",
-		"json.c.0":       "1",
-		"json.c.1":       "2",
-		"json.c.2":       "3",
-		"json.d.a.b":     "1",
-		"json.e.0.a":     "1",
-		"json.f.0.0.0.z": "abc",
-	}
-	jsonMap, err := jsonToMap([]byte(jsonMapString))
-	if err != nil {
-		t.Error(err)
-	}
+]`,
+		want: map[string]string{
+			"json":             "2",
+			"json.0":           "1",
```

Comment:
Previously array of array wasn't adding the length keys but I think it's supposed to